app/assets/javascripts/users/profile.js

```diff
@@ -19,4 +20,8 @@
     // Will change this during proper implementation.
     $(".donations-bars").css("padding-right", (($(".donations-bars > div").length -1)* 1) + "px");  
   } 
+
+  function flashFadeOut(){
+    $(".alert-success").delay(2200).slideUp(1000);
```

Comment:
nitpick - probably good to check if `.alert-success` exists before executing the flash hide behavior.

Also this is more of a nitpick, javascript people hate using double quotes lol

Reply:
lol double feels more natural to me because of Ruby. I can change it. But then I'll have to change it in every js file. Maybe we should add a js linter?

Reply:
yeah linter is changing everything to double quotes in the ruby files, so fine to leave it as is by me. You'll want to run `rails check` before checking in. I don't have a JS linter on there though, I'll add a card for that.

Reply:
I ran the check task. I fixed everything(except for the erb complaints) rubocop complained about before making the pr. I didn't fix the erb complaints because there were so many of them. Pretty much in every file. We can have a separate task to fix those.

Reply:
ok cool, yeah thought all those erb warnings could be done as a good warm up task for some of the newcomers. thanks